### PR TITLE
Minor tweaks for pkgdown; remove vignettes `abstract` element

### PR DESCRIPTION
**Pull request by amanda-hi**
- removed "Abstract" section from vignettes, as this is covered by the vignette "Introduction" section
- removed SomaLogic logo from README
- updated NEWS file to reflect addition of pkgdown site

### CODE REVIEW

**Comment by stufield** [comment]
Just to note, the `*.png` in the PR here all have (TM) logo, rather than the (R) logo.
To update with the new correct logo, run:
```
Rscript --vanilla -e "pkgdown::build_favicons()"
```
See:
https://pkgdown.r-lib.org/reference/build_favicons.html

File: README.Rmd [hunk 1]
```diff
@@ -19,7 +19,7 @@ ver <- paste0("https://img.shields.io/badge/Version-", ver,
 ```
 
 
-# `SomaScan.db` <img src="inst/figures/logo.png" align="right" height="138" alt="" />
+# `SomaScan.db`
```
Comment (stufield) [inline README.Rmd]: If you remove this from here (even though BioC doesn't like it), won't the logo will be missing from the GH README page?

**Review by stufield — APPROVED** [review]
Looks like the favicons need to be updated to the new logo with the (R) ... see other comment.

File: README.Rmd (hunk 1 above)
Comment (amanda-hi) [reply README.Rmd]: Yes, it does end up being removed from the GH README, but if I leave the image reference there, I get a broken image icon next to the package title in the pkgdown site. It seems like the only way to satisfy both situations (GH README _and_ pkgdown landing page) is to remove the image, unfortunately.

Comment (amanda-hi) [reply README.Rmd]: Here is a screenshot of the pkgdown site, for reference: image

File: _pkgdown.yml [hunk 2]
```diff
@@ -22,8 +22,6 @@ footer:
   structure:
     left: [package, developed_by]
     right: built_with
-  components:
```
Comment (stufield) [inline _pkgdown.yml:25]: I would leave this in and have it look like this:
```
components:
  logo: "<img src='favicon-16x16.png'>"
```
but do *not* add the `logo` element to the `left: [package, developed_by]` section. 
this way it doesn't do anything here, but in the `pkgdown` branch, you can add `logo` to the `left:` component, and it'll take over from there. (i.e. less change between branches ... less to get our of sync).

**Review by stufield — APPROVED** [review]

**Comment by stufield** [comment]
Could you please add `Canopy (Python)` to the link in L44 of `_pkgdown.yml` to match the other repos?

File: _pkgdown.yml (hunk 2 above)
Comment (amanda-hi) [reply _pkgdown.yml:25]: This is a good idea!

**Pull request merged**